recipes/inn-interface/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "INN-interface" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com//NREL/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 5ca5617ec9c7315de72fabd705f708f3515877c507335571d2f5dac8964dab8a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
```

Comment:
https://github.com/NREL/INN-interface/blob/ab4be7cdae5c1d1329c1e78636c0556eaa512f74/setup.py#L13

Reply:
Hey @carterbox, I just checked and the files in examples and models folder are not needed for the module. I also updated the python version to match the setup.py. Thanks for catching that!